src/components/App.jsx

```diff
@@ -5,52 +5,76 @@ import Header from './Header'
 import Prompt from './Prompt'
 import prompts from '../data/prompts.json'
 import History from './History'
+import loadFromLocal from '../lib/loadFromLocal'
+import saveToLocal from '../lib/saveToLocal'
 
 function App() {
+  const INITIALPROMPT = 'Spin to receive your first prompt.'
+  const LASTPROMPT = `The Wheel is tired.
+  No more spins until you reset.`
+
   const [currentPrompt, setCurrentPrompt] = useState(
-    'Spin to receive your first prompt.'
+    loadFromLocal('currentPrompt') ?? INITIALPROMPT
   )
-  const [history, setHistory] = useState([])
+  const [history, setHistory] = useState(loadFromLocal('promptHistory') ?? [])
 
   return (
     <Grid>
       <Header>Wheel of TBR</Header>
       <Main>
-        <Prompt>{currentPrompt}</Prompt>
+        <Prompt data-testid="prompt">{currentPrompt}</Prompt>
         <FlexWrapper>
           <Button
-            disabled={currentPrompt.includes('Wheel is tired')}
+            disabled={currentPrompt.includes(LASTPROMPT)}
             primary
-            onClick={setRandomPrompt}
+            onClick={onSpin}
           >
             Spin!
           </Button>
+          <Button
+            disabled={currentPrompt.includes(INITIALPROMPT)}
+            onClick={onReset}
+          >
+            reset
+          </Button>
         </FlexWrapper>
         {history.length ? <History history={history} /> : ''}
       </Main>
     </Grid>
   )
 
-  function setRandomPrompt() {
+  function onReset() {
+    setHistory([])
+    saveToLocal('promptHistory', [])
+    setCurrentPrompt(INITIALPROMPT)
+    saveToLocal('currentPrompt', INITIALPROMPT)
+  }
+
+  function onSpin() {
+    let randomPrompt = prompts[getRandomNumber()]
+
     function getRandomNumber() {
```

Comment:
You might want to source out this relatively long function to a separate file, to keep your App.js more clear?

Reply:
good idea!! Thank you very much! 😊